include/sys/zfs_context.h

```diff
@@ -32,7 +32,7 @@
 extern "C" {
 #endif
 
-#ifdef __KERNEL__
+#if defined(__KERNEL__) || defined(_STANDALONE)
```

Comment:
Comments on the #else and #endif should match.

Reply:
will do these and others highlighted...